src/Arcus.Messaging.Abstractions/Arcus.Messaging.Abstractions.csproj

```diff
@@ -29,7 +29,10 @@
   <ItemGroup>
     <PackageReference Include="Arcus.Observability.Telemetry.Serilog.Enrichers" Version="[3.0.0,4.0.0)" />
     <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.21.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+
+    <!-- TODO: will be removed in v3.0 -->
+    <PackageReference Include="System.Text.Json" Version="[8.0.5,10.0.0)" Condition="'$(TargetFramework)' != 'net8.0'" />
+    <!-- TODO: will be removed in v3.0 -->
```

Comment:
What do you mean ?  Why will we remove this dependency in a later version ?

Reply:
As it is only used by .NET Standard and.NET 6. And we are migrating away from them, I believe.

Reply:
It's included by default 8n .NET 8, but not in the other frameworks.

Reply:
But, should we then have a version range 'til 10.x ?

Reply:
Think Microsoft uses major versions not in the normal sense, but more in line with their .NET versions. Think therefore that .NET 9 is reflected here, instead of being a breaking change version.

Reply:
Bc I think that (just like from .NET 6 to .NET 8), there are breaking changes. Unlike from .NET 6 to .NET 7, believe.

Reply:
Hmm, ok.
But then again, we only need this package when we target < .NET 8.
So, in that case, I think I would change the condition and specify that we only include this package when we're targetting .netstandard.
And in that case, I think it would be sufficient to specify that we want `Version=8.*` ?

Might be a bit clearer ?  Not that important anyway as we'll remove this in a later version

Reply:
* We also need it (I saw) in .NET 6, as there is a missing feature for 'missing JSON properties' that is not available in the built-in version of .NET 6;
* And fine for me to do it instead with wildcards.

Reply:
And version 8.0.5 works for .NET 6 ?

Reply:
Ran the tests with both frameworks successfully, so I would guess so, yes.